tests/components/overkiz/test_config_flow.py

```diff
@@ -144,10 +141,9 @@ async def test_options_flow(hass):
     )
 
     with patch("pyhoma.client.TahomaClient.login", return_value=True), patch(
-        "homeassistant.components.overkiz.async_setup", return_value=True
-    ), patch("homeassistant.components.overkiz.async_setup_entry", return_value=True):
+        "homeassistant.components.overkiz.async_setup_entry", return_value=True
+    ):
         entry.add_to_hass(hass)
-        assert await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
I think you still need this

Reply:
Sorry for the back and forth notifications on this PR. I will see if I can easily run these tests again in my devcontainer first... :-)

Reply:
You are right, this was the fix. Local tests did pass now.

Reply:
```
Results (0.29s):
       1 passed
       9 deselected
root@ha-dev:~/home-assistant# pytest -vvvs -k test_options_flow tests/components/overkiz/test_config_flow.py
```
LGTM